docs/operations/updating_the_stack.rst

```diff
@@ -79,178 +94,134 @@ You can copy down your ``cops.pem`` into your ``~/.ssh`` from bastion2 by:
 .. note:: Example above assumes that a copy of ``cops.pem`` for  **IdentityFile** is copied to where your ssh keys are.
 
 
-
 ----
 
-*****
-Steps
-*****
+***********************
+COPS Stack Deploy Steps
+***********************
 
-1. Update your buildout and JS dependencies
+0. Update Buildout and JS Dependencies
 ===========================================
 
-Keep sure you are on the latest master branch.
+Make sure you are on the latest master branch.
+
+.. code-block:: bash
+
+   $ cd output-producer-service
+   $ git checkout master
```

Comment:
We haven't had to worry about this too because our merge velocity is fairly light, but technically (to be safe), it's best to checkout the specific git ref associated with the `output-producer-service` repo in the build of the code version tag selected for a deployment. Is it worth making this clear as part of these updates?

Reply:
yes it is, will do - thank you!